providers/flagd/pkg/configuration.go

```diff
@@ -0,0 +1,116 @@
+package flagd
+
+import (
+	"fmt"
+	"github.com/go-logr/logr"
+	"github.com/open-feature/go-sdk-contrib/providers/flagd/pkg/cache"
```

Comment:
I think these are not formatted correctly

Reply:
We do not enforfce gofumpt here. I have run intellij formatter and this is the output it gives 🤔  I think this is simple alphabetic ordering

Reply:
you are right 👍